test/lan_items_helpers_test.rb

```diff
@@ -590,3 +565,113 @@ def mock_dhcp_setup(ifaces, global)
     end
   end
 end
+
+describe "LanItems renaming methods" do
+  let(:renamed_to) { nil }
+  let(:current) { 0 }
+  let(:item_0) do
+    {
+      "ifcfg"      => "eth0",
+      "renamed_to" => renamed_to,
+      "udev"       => {
+        "net" => [
+          "ATTR{address}==\"24:be:05:ce:1e:91\"",
+          "NAME=\"#{renamed_to}\""
```

Comment:
I'd prefer to mock relevant methods bcs for ```renamed_to == nil``` this creates an invalid udev rule and also in theory it can be difference if ```renamed_to``` key is not present or ```{renamed_to => nil}``` 

I know that currently it is not such difference, but  you're mocking by defining an internal structure which hopefully will be subject of change *soon* instead of mocking return values of several already well known methods.

Reply:
That is true, but basically I moved your mock from

https://github.com/yast/yast-network/blame/master/test/network_autoyast_test.rb#L394

Reply:
that doesn't mean that it is correct. My tests are usually .... not so perfect ;-)